README.md

```diff
@@ -1,22 +1,28 @@
 # Uber Rides Java SDK (Beta) [![Build Status](https://travis-ci.org/uber/rides-java-sdk.svg?branch=master)](https://travis-ci.org/uber/rides-java-sdk)
 This SDK helps your Java App make HTTP requests to the Uber Rides API.
 
+## Prerequisites
+- Dependency manager: Gradle or Maven
```

Comment:
Well, Ivy or other custom dep managers could technically resolve this as well. It just feels noisy to me and I haven't really seen this called out in other sdk's and libraries. Can you please clarify what you meant by "devs trying to use the SDK because there's no one available"?

OkHTTP has a good example of Gradle or Maven instructions https://github.com/square/okhttp